source/documentation/admin-guide/appe-oVirt_and_SSL.html.md

```diff
@@ -92,15 +92,27 @@ The internal CA stores the internally generated key and certificate in a **P12**
         SSL_CERTIFICATE=/etc/pki/ovirt-engine/certs/apache.cer
         SSL_KEY=/etc/pki/ovirt-engine/keys/apache.key.nopass
 
-10. Restart the `ovirt-provider-ovn` service:
+10. Edit the **/etc/ovirt-imageio-proxy/ovirt-imageio-proxy.conf** 
+        
+        # vi /etc/ovirt-engine/ovirt-websocket-proxy.conf.d/10-setup.conf
+
+   Make the following changes and save the file:
```

Comment:
You need to indent this line like the similar line in section 9, so it will be considered as part of section 10.

Reply:
I guess I don't see the difference here - I copied 9 to make this entry.  What am I missing?

Reply:
I think there are 4 spaces before "Make the following changes and save the file:" in section 9, and only 3 in section 10.

Reply:
Thanks, found the issue was my indentation was set to 3 spaces, should have been 4.